index.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let data = document.querySelector(".data");
+    let columnOne = document.querySelector(".columnOne");
+    let columnTwo = document.querySelector(".columnTwo");
+    let getPokeBtn = document.querySelector("#pokeBtn");
+    let battleBtn = document.querySelector("#battleBtn");
+    const randomPoke = async (event) => {
```

Comment:
You can have an argument here to append to a specific div, this would decrease the code you have by half.